src/main/g8/.travis.yml

```diff
@@ -13,7 +13,10 @@ cache:
   - \$HOME/.sbt/boot/
 
 script:
-  - sbt ++\$TRAVIS_SCALA_VERSION -jvm-opts .jvmopts clean test &&
-    sbt coverageAggregate
+  - sbt ++\$TRAVIS_SCALA_VERSION -jvm-opts .jvmopts clean coverage test
+
+
 after_success:
-  - sbt coveralls
+  - sbt coverageReport
+  - sbt coverageAggregate
+  - sbt codacyCoverage
```

Comment:
this is how they recommended. I remember that coverage had issue with running aggregate and report in the same sbt session. but let me try run it in one session in kanaloa.